lib/index.js

```diff
@@ -19,6 +20,12 @@ var KISSmetrics = module.exports = integration('KISSmetrics')
   .retries(2);
 
 /**
+ * Use a POST request if the GET query string payload exceeds 2KB.
+ */
+
+var MAX_GET_PAYLOAD = 1 << 11;
```

Comment:
I think for the kb calculatation `<< 10` is definitely more readable and more commonly used. `2 << 10` is easy to read as 2kb. `1<<11` gives me, well nothing.